docs/examples/sock-shop.md

```diff
@@ -117,18 +151,18 @@ and if we visit `http://localhost:8080`, we'l see:
 Pretty simple! Now, let's go back and look at that command in more detail:
 
 ```console
-gitops add app \                                                   # (1)
+gitops add app \                                                 # (1)
    --url ssh://git@github.com/example/microservices-demo.git \   # (2)
    --path ./deploy/kubernetes/manifests \                        # (3)
-   --app-config-url NONE                                         # (4)`
+   --app-config-url ssh://git@github.com/example/external.git    # (4)
+   --auto-merge                                                  # (5)`
 ```
 
 1. Add an application to a cluster under the control of Weave GitOps
 2. The application is defined in the GitHub repository at the specified URL
 3. Only the manifests at the specified path within the repository are part of the application
-4. Don't store the management manifests in GitHub; the `app-config-url` parameter says where to store management manifests. The default location (if no `app-config-url` is specified) is to place them in a hidden directory (`.wego`) within the application repository itself. An actual URL value causes them to be stored in the repository referenced by the URL. `NONE` means to apply them to the cluster but don't store them in `git` at all.
-
-For a quick turnaround (during development or for testing) `NONE` does the trick. The application is being managed via GitOps, so any changes made in the application repository will be applied to the cluster when they are merged.
+4. Store the management manifests in a separate configuration repository within GitHub; the `app-config-url` parameter says where to store management manifests. The default location (if no `app-config-url` is specified) is to place them in the `.weave-gitops` directory within the application repository itself. An actual URL value causes them to be stored in the repository referenced by the URL
+5. Don't create a pull request for the management manifests; push them directly to the upstream repository
 
 The application can also be deployed via a helm chart. Applications defined in helm charts can be deployed from either helm repositories or git repositories. In the case of the *Sock Shop* application, a helm chart is included in the `GitHub` repository. We only need to make minor changes to the command we used above to switch to a helm chart, but using a helm chart for *Sock Shop* requires the target namespace to exist before deploying. By default, the chart would be deployed into the `wego-system` namespace (since we know it exists), but we'd like to put it in the `sock-shop` namespace. So, before we run `gitops add app`, we'll run:
```

Comment:
Should this have a header breaking into a new section?

Reply:
Moved to new "Using Helm Charts" section